python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -313,11 +313,15 @@ async def monitor_position(self, check=None):
                 )
 
             if _check.mtrotator:
-                rotator = await self.rem.mtrotator.tel_application.next(
+                rotator = await self.rem.mtrotator.tel_rotation.next(
```

Comment:
Please consider `rotation` or `rotation_data` for the data. It is a better match to the telemetry topic name.

(In a unit test you use a completely different variable name for the same thing, which I complain about below).